cypress/e2e/modal-overlays.spec.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+
+describe('Modal&Overlays', ()=>{
+    
+
+    it('Hover the element without clicking it', ()=> {
+        cy.visit('/pages/modal-overlays/popover').wait(3000)
```

Comment:
```suggestion
        cy.visit('/pages/modal-overlays/popover')
```
вважається поганою практикою вставляти явні очікування в тест